packages/smooth_app/lib/pages/product/nutrition_page/widgets/nutrition_facts_editor.dart

```diff
@@ -343,6 +343,10 @@ class _NutrientValueCell extends StatelessWidget {
                               value == 'traces') {
                             return null;
                           }
+                          // special case: "missing" nutrient
+                          if (value == _missingNutrientValue) {
+                            return null;
```

Comment:
don't we have support for - in product opener ?

Reply:
@teolemon Modifiers aren't fully implemented in off-dart for the moment.
We can have Robotoff extract several modifiers ('~', '>' and '<'), not including '-'.
We cannot save product nutrients with modifiers anyway.
* https://github.com/openfoodfacts/openfoodfacts-dart/issues/584
* https://github.com/openfoodfacts/openfoodfacts-dart/pull/1042
* https://github.com/openfoodfacts/smooth-app/issues/3135

Reply:
@teolemon your question could be interpreted in two different ways, I think - I'll answer the remaining one just in case -- this is a validation function, and `return null;` indicates that the input is valid/accepted.  So this logic states "if the user input is `-`, it's OK to submit to the server".

(@monsieurtanuki I hope that's correct; that's the way I'd understood the code during code review)